vividus-tests/src/main/resources/story/integration/WaitStepsTests.story

```diff
@@ -11,3 +11,12 @@ Then number of elements found by `${paragraph-locator}` is equal to `1`
 When I click on a button with the name 'Hide'
 When I wait until element located `${paragraph-locator}` disappears
 Then number of elements found by `${paragraph-locator}` is equal to `0`
+
+Scenario: Step verification I wait '$duration' with '$pollingDuration' polling until an element located by $locator disappears
+Given I am on a page with the URL 'https://www.w3schools.com/jQuery/tryit.asp?filename=tryjquery_hide_slow'
```

Comment:
would you mind adding the page to https://github.com/vividus-framework/vividus-test-site?